Automattic/wp-calypso
# Replace `createReducer[WithValidation]` across Calypso

This PR was generated from the following codemod: Automattic/calypso-codemods#8

The code it produces appears to be sane, and there are no instances of the possible TODO introduced by the codemod to be manually handled.

I also applied some manual changes to improve generated code, where linting issues popped up.

All Calypso unit tests continue to work after this mod.

#### Changes proposed in this Pull Request

* Replace most usage of `createReducer` and `createReducerWithValidation` in the codebase, after the deprecation in #35905. 

#### Testing instructions

This touches a lot of state, so it's difficult to provide detailed testing instructions. Use as much of Calypso as you feel like, and see if anything goes wrong.

I also enabled the full suite of E2E tests to provide some measure of assurance.
